src/Symfony/Component/Translation/TranslatorInterface.php

```diff
@@ -20,6 +20,8 @@
  */
 interface TranslatorInterface
 {
+    const DEFAULT_DOMAIN = 'messages';
```

Comment:
`public const`? What about moving it to `MessageCatalogue` instead as a constructor arg?